include/dlr_pipeline.h

```diff
@@ -0,0 +1,66 @@
+#ifndef DLR_PIPELINE_H_
+#define DLR_PIPELINE_H_
+
+#include <graph/graph_runtime.h>
+#include <tvm/runtime/memory.h>
+#include "dlr_common.h"
+
+#if defined(_MSC_VER) || defined(_WIN32)
+#define DLR_DLL __declspec(dllexport)
+#else
+#define DLR_DLL
+#endif  // defined(_MSC_VER) || defined(_WIN32)
+
+namespace dlr {
+
+/*! \brief class PipelineModel
+ */
+class DLR_DLL PipelineModel : public DLRModel {
```

Comment:
Nitpick: Instead of PipelineModel, should we call this "ModelPipeline" which in more representative of the underlying implementation and communicates whats actually happening.

Reply:
Design doc says that in addition to `CreateDLRModel` we need new method `CreateDLRPipeline`.
Since all `DLRModel` classes ends with Model I named Pipeline class `PipelineModel`